apps/davi/src/stores/modules/subgraph/common/useGuildConfig/index.ts

```diff
@@ -0,0 +1,125 @@
+import { BigNumber } from 'ethers';

```

Comment:
should we have this on a useGuildConfig.ts and export it from index?

Reply:
Could you elaborate on this? Do you mean that the folder structure should look different?

Reply:
yes, i mean having this: 

+ subgraph
|-+ common
| |-+ useGuildConfig 
| | |- index.ts
| | |- useGuildConfig.ts
| | | |- query.graphql

let me know if i'm not being clear